index.d.ts

```diff
@@ -0,0 +1,760 @@
+// Definitions by James George (https://github.com/jamesgeorge007)
+
+export function cli(e: any): any;
+
+export function main(e: any, t: any): any;
+
+interface Config {
+	System: string[];
```

Comment:
Would be great if we have type hints about the available helpers, that way we don't have to read the docs but simply check the autocompletion whats available 😄 

I think it's fairly easy to set this up, but it requires some changes. if we define an interface for the helper consts, like `browserBundleIdentifiers`, we can do something like:

```
interface Config {
    Browsers: (keyOf helpers.BrowserBundleIdentifiers)[];
}
```